docs/ocis/development/testing.md

```diff
@@ -376,6 +376,48 @@ BEHAT_FEATURE="tests/acceptance/features/apiNotification/emailNotification.featu
 make test-acceptance-api
 ```
 
+## Running Test Suite With Tika Service (@tikaServiceNeeded)
+
+Test suites that are tagged with `@tikaServiceNeeded` require tika service.
+
+### Setup Tika Service
+
+Run the following docker command to setup tika service
+
+```bash
+docker run -d -p 127.0.0.1:9998:9998 apache/tika
+```
+
+### Run oCIS
+
+Documentation related to the content based search and tika extractor can be found [here](https://doc.owncloud.com/ocis/next/deployment/services/s-list/search.html#content-extraction)
+
+```bash
+# init oCIS
+IDM_ADMIN_PASSWORD=admin \
+ocis/bin/ocis init --insecure true
+
+# run oCIS
+PROXY_ENABLE_BASIC_AUTH=true \
+OCIS_INSECURE=true \
+SEARCH_EXTRACTOR_TYPE=tika \
+SEARCH_EXTRACTOR_TIKA_TIKA_URL=http://localhost:9998 \
+SEARCH_EXTRACTOR_CS3SOURCE_INSECURE=true \
```

Comment:
FRONTEND_FULL_TEXT_SEARCH_ENABLED=true \
Is this needed or not?

Reply:
i thinks its only needed for web UI not for API tests

Reply:
yeah, needed for web client